### ISSUE TITLE
App crash/close when tapping on "Add Integration" button

### ISSUE THREAD

**Issue by Maxouwell** [home-assistant/android#4761]
**Home Assistant Android app version(s):**
2024.10.3-full
**Android version(s):**
14
**Device model(s):**
Pixel 7a
**Home Assistant version:**
Core 2024.10.4
Supervisor 2024.10.3
Operating System 13.2
Interface utilisateur 20241002.4
**Last working Home Assistant release (if known):**

**Description of problem, include YAML if issue is related to notifications:**
When I tap on the Add Integration button on the "Devices and Services" section of the parameters, the app close and seems to crash.
Before, I added successfully another integration (the MeteoFrance one) and deleted the default weather integration (I don't remember it's name).

**Companion App Logs:**
Logs are after restart/reopen and don't seems to indicate the problem.
```
--------- beginning of main
10-28 19:18:24.145 27070 27070 D WebviewActivity: Matter/Thread step changed to NOT_STARTED
10-28 19:18:24.157 27070 27244 D WebviewActivity: External bus {"type":"connection-status","payload":{"event":"disconnected"},"id":8}
10-28 19:18:24.160 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.162 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.162 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139433691, currentMillis: 1730139504162)
10-28 19:18:24.296 27070 27237 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.297 27070 27126 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=da5f87d2-aa43-4808-b64e-cb865646b943, tags={ io.homeassistant.companion.android.websocket.WebsocketManager } ]
10-28 19:18:24.300 27070 27126 I WM-Processor: Moving WorkSpec (c0579f65-3473-42c5-a5e1-fb7ddf7491b5) to the foreground
10-28 19:18:24.313 27070 27070 I WM-SystemFgDispatcher: Started foreground service Intent { act=ACTION_START_FOREGROUND cmp=io.homeassistant.companion.android/androidx.work.impl.foreground.SystemForegroundService (has extras) }
10-28 19:18:24.315 27070 27236 D SensorWorker: Updating all Sensors in foreground.
10-28 19:18:24.322 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.322 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139433691, currentMillis: 1730139504322)
10-28 19:18:24.323 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.335 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.335 27070 27070 D ServerConnectionInfo: Using external URL
10-28 19:18:24.337 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.337 27070 27070 D ServerConnectionInfo: Using external URL
10-28 19:18:24.346 27070 27244 D WebviewActivity: External bus {"type":"connection-status","payload":{"event":"connected"},"id":9}
10-28 19:18:24.365 27070 27070 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.378 27070 27237 D LocBroadcastReceiver: Background location updates appear to have stopped, restarting location updates
10-28 19:18:24.378 27070 27237 D LocBroadcastReceiver: Removing background location requests.
10-28 19:18:24.381 27070 27237 D LocBroadcastReceiver: Registering for location updates.
10-28 19:18:24.399 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.401 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:24.416 27070 27194 D TrafficStats: tagSocket(153) with statsTag=0xffffffff, statsUid=-1
10-28 19:18:24.417 27070 27244 D WebviewActivity: External bus {"type":"theme-update","id":10}
10-28 19:18:24.424 27070 27070 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
10-28 19:18:24.424 27070 27070 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
10-28 19:18:24.424 27070 27236 D WebViewPresenterImpl: Try getting color from webview color "#101e24".
10-28 19:18:24.424 27070 27236 I WebViewPresenterImpl: Found color -15720924.
10-28 19:18:24.424 27070 27237 D WebViewPresenterImpl: Try getting color from webview color "#111111".
10-28 19:18:24.424 27070 27237 I WebViewPresenterImpl: Found color -15658735.
10-28 19:18:24.470 27070 27236 D SensorReceiver: Nothing to update for server 3 (Maison)
10-28 19:18:24.471 27070 27236 I SensorReceiver: Sensor updates and sync completed
10-28 19:18:24.472 27070 27127 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=c0579f65-3473-42c5-a5e1-fb7ddf7491b5, tags={ io.homeassistant.companion.android.sensors.SensorWorker } ]
10-28 19:18:24.474 27070 27070 I WM-SystemFgDispatcher: Stopping foreground service
10-28 19:18:28.276 27070 27244 D WebviewActivity: External bus {"type":"config_screen/show","id":11}
10-28 19:18:28.297 27070 27070 D IntegrationRepository: setAppActive(): false
10-28 19:18:28.297 27070 27070 D IntegrationRepository: setAppActive(): false
10-28 19:18:28.302 27070 27070 D SensorReceiver: Received intent: io.homeassistant.companion.android.UPDATE_SENSORS
10-28 19:18:28.304 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:28.326 27070 27115 D LocBroadcastReceiver: Background location updates appear to have stopped, restarting location updates
10-28 19:18:28.326 27070 27115 D LocBroadcastReceiver: Removing background location requests.
10-28 19:18:28.330 27070 27115 D LocBroadcastReceiver: Registering for location updates.
10-28 19:18:28.343 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:28.345 27070 27236 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:28.377 27070 27070 I ContentCaptureHelper: Setting logging level to OFF
10-28 19:18:28.379 27070 27236 D SensorReceiver: Nothing to update for server 3 (Maison)
10-28 19:18:28.379 27070 27236 I SensorReceiver: Sensor updates and sync completed
10-28 19:18:28.379 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139433691, currentMillis: 1730139508379)
10-28 19:18:28.433 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.433 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.434 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:28.438 27070 27115 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: true, usesWifi is: true
10-28 19:18:28.453 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:28.455 27070 27070 D SettingsWearDetection: API unavailable for discovering nodes (no Wear)
10-28 19:18:28.588 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139433691, currentMillis: 1730139508588)
10-28 19:18:28.588 27070 27070 D IntegrationRepository: setAppActive(): true
10-28 19:18:28.665 27070 27711 D TrafficStats: tagSocket(298) with statsTag=0xffffffff, statsUid=-1
10-28 19:18:28.953 27070 27070 D VRI[WebViewActivity]: visibilityChanged oldVisibility=true newVisibility=false
10-28 19:18:28.973 27070 27070 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=androidx.activity.OnBackPressedDispatcher$Api34Impl$createOnBackAnimationCallback$1@4e3cf26
10-28 19:18:29.353 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.800 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.813 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.832 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.847 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.864 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.884 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.897 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.918 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.931 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.948 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.970 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.970 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.980 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:29.996 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.018 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.019 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.030 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.048 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.066 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.066 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.080 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.097 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.116 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.130 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.148 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.165 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.181 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.197 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:30.215 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:18:33.241 27070 27070 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
10-28 19:18:34.922 27070 27070 D CompatibilityChangeReporter: Compat change id reported: 147798919; UID 10346; state: ENABLED
10-28 19:18:34.941 27070 27070 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.widget.PopupWindow$PopupDecorView$$ExternalSyntheticLambda1@fbb1ae9
10-28 19:18:34.990 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: true, expireMillis: 1730139433691, currentMillis: 1730139514990)
10-28 19:18:34.990 27070 27070 D IntegrationRepository: setAppActive(): true
10-28 19:18:35.258 27070 27130 D HWUI    : endAllActiveAnimators on 0xb4000072f13dc260 (MenuPopupWindow$MenuDropDownListView) with handle 0xb400007191302330
10-28 19:18:35.928 27070 27070 D IntegrationRepository: setAppActive(): false
10-28 19:18:35.931 27070 27070 D IntegrationRepository: setAppActive(): false
10-28 19:18:35.954 27070 27070 D VRI[SettingsActivity]: visibilityChanged oldVisibility=true newVisibility=false
10-28 19:19:51.935 27070 27244 D WebviewActivity: External bus {"type":"connection-status","payload":{"event":"disconnected"},"id":12}
10-28 19:19:51.937 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139516928, currentMillis: 1730139591937)
10-28 19:19:51.989 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:51.990 27070 27070 E mpanion.android: Invalid resource ID 0x00000000.
10-28 19:19:52.022 27070 27070 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1730139516928, currentMillis: 1730139592022)
10-28 19:19:52.022 27070 27070 D IntegrationRepository: setAppActive(): true
10-28 19:19:53.125 27070 27070 D IntegrationRepository: setAppActive(): false
```

**Screenshot or video of problem:**

**Additional information:**

**Comment by jpelgrom** [comment]
Would you be willing to try the beta version of the app? There is a fix for a potential crash when adding devices in there.

**Comment by Maxouwell** [comment]
Yes, I'll try to activate it in the play store

**Comment by Maxouwell** [comment]
The beta version 2024.10.4-full fixed the problem, thanks, and sorry if I missed a bug report while searching.

**Comment by jpelgrom** [comment]
There actually wasn't a bug report/issue for it, only a fix once it showed up in automatically reported crashes (low number of people, but I guess you're one of the unlucky few).

**Comment by jpelgrom** [comment]
My assumption is that it was fixed in #4740
